components/Cards/examples/CardsCollectionTintingExample.m

```diff
@@ -75,6 +83,22 @@ - (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView
                                                 forIndexPath:indexPath];
   [cell setTintColor:[UIColor redColor]];
   [cell setNeedsDisplay];
+
+  // enable interactable mode and accessibility for interactivity.
+  if (_interactable) {
+    cell.interactable = YES;
+    cell.selectable = YES;
+
+    cell.isAccessibilityElement = YES;
+    cell.accessibilityCustomActions = @[
+      [[UIAccessibilityCustomAction alloc] initWithName:@"Toggle Switch"
```

Comment:
Nit: This should be indented 2 more spaces ([total 4](http://go/objc-style#Spacing_and_Formatting)) for a statement continuation.